test/integration/Android/Session/LogTests.cs

```diff
@@ -32,6 +35,20 @@ public void TearDown()
             _driver.Dispose();
         }
 
+        [Test]
+        public async Task AAA()
+        {
+            //OpenQA.Selenium.Internal.Logging.Log.SetLevel(OpenQA.Selenium.Internal.Logging.LogEventLevel.Trace);
+
+            await using var bidi = await _driver.AsBiDiAsync();
+
+            Console.WriteLine(await bidi.StatusAsync());
+
+            await bidi.Log.OnEntryAddedAsync(Console.WriteLine, new() { Contexts = [new(bidi, "NATIVE_APP")] });
+
+            await Task.Delay(5000);
+        }
+
```

Comment:
We already have a test file for BiDi.
A better location for it will be there IMO.
Or a new file for BiDi Logs under BiDi folder.